### PR TITLE
Create Proto Ble Project

### PR DESCRIPTION
**Pull request by matthewcarlreetz**

### CODE REVIEW

File: packages/proto-ble/package.json [hunk 1]
```diff
@@ -0,0 +1,40 @@
+{
+  "name": "proto-ble",
+  "version": "3.9.1",
```
Comment (tyler-whitman) [inline packages/proto-ble/package.json:3]: Should we start it at 1.0.0?

Comment (allenan) [reply packages/proto-ble/package.json:3]: I think it'll get autolinked to the versions of the others once I do the first publish. So shouldn't really matter

**Review by tyler-whitman — APPROVED** [review]

File: packages/proto-ble/package.json (hunk 1 above)
Comment (tyler-whitman) [reply packages/proto-ble/package.json:3]: Ok np

File: packages/proto-ble/src/__tests__/gateway.test.ts [hunk 2]
```diff
@@ -0,0 +1,11 @@
+import { add_gateway_v1 as AddGatewayV1, Iadd_gateway_v1 } from '../../build'
+
+const dict: Iadd_gateway_v1 = { payer: 'tacos mcgee' }
+describe('wifi_remove_v1', () => {
```
Comment (allenan) [inline packages/proto-ble/src/__tests__/gateway.test.ts:4]: i think this `describe` got copied/pasted around

File: packages/proto-ble/package.json [hunk 3]
```diff
@@ -0,0 +1,40 @@
+{
+  "name": "proto-ble",
```
Comment (allenan) [inline packages/proto-ble/package.json:2]: name should be `"@helium/proto-ble"`

```suggestion
  "name": "@helium/proto-ble",
```

**Review by allenan — CHANGES_REQUESTED** [review]
overall looks great! just a couple minor things then this can be merged and published

**Review by allenan — APPROVED** [review]
lg

**Pull request merged**